release/src/main/scripts/run_rc_validation.sh

```diff
@@ -648,7 +646,7 @@ if [[ ("$python_xlang_kafka_taxi_dataflow" = true
   do
     rm -rf ./beam_env_${py_version}
     echo "--------------Setting up virtualenv with $py_version interpreter----------------"
-    virtualenv beam_env_${py_version} -p $py_version
+    $py_version -m venv beam_env_${py_version}
```

Comment:
Same question about the release script changes, but fortunately you are the next release manager, so I trust you'll be able to detect and fix any issues here.